maintenance_plan/migrations/13.0.1.2.4/post-migration.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# pylint: disable=C7902
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr, "maintenance_plan", "migrations/13.0.1.2.0/noupdate_changes.xml"
```

Comment:
You need to change also the filename here!

Reply:
https://github.com/OCA/maintenance/pull/210